app/routes/circulars._archive._index/route.tsx

```diff
@@ -268,7 +281,7 @@ export default function () {
             name="query"
             type="search"
             defaultValue={inputQuery}
-            placeholder="Search"
+            placeholder={searchText}
```

Comment:
This is also implemented in #2732, with slightly different wording.

Reply:
which one would we like to go with? Would you like me to remove the placeholder in this branch?